ci/build_libtelio.py

```diff
@@ -281,6 +281,10 @@ def main() -> None:
 
     args = parser.parse_args()
 
+    if "true" not in [os.getenv("GITLAB_CI"), os.getenv("GITHUB_ACTIONS")]:
+        if "BYPASS_LLT_SECRETS" not in os.environ:
```

Comment:
I think it's worth to have case-insensitive checks

Reply:
I don't think so, CI sets "true" with low cap, and environment variables should be set as capital letters.